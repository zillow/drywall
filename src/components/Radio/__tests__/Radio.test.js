import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Radio } from '../../../index';

describe('<Radio>', () => {
    it('renders a default radio', () => {
        const testRenderer = TestRenderer.create(<Radio name="test-radio" />);
        const tree = testRenderer.toJSON();
        expect(tree.type).toBe('input');
        expect(tree.props.type).toBe('radio');
        expect(tree.props.name).toBe('test-radio');
        expect(tree).toMatchSnapshot();
    });
});
