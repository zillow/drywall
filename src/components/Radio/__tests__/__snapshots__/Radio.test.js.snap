// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio> renders a default radio 1`] = `
<input
  name="test-radio"
  type="radio"
/>
`;
